ruby/red-arrow/test/test-table.rb

```diff
@@ -1049,27 +1049,29 @@ def setup
                                 number: [10, 20, 30])
       table2 = Arrow::Table.new(key: [3, 1],
                                 string: ["three", "one"])
-      assert_equal(Arrow::Table.new([
-                                      ["key", [1, 3]],
-                                      ["number", [10, 30]],
-                                      ["key", [1, 3]],
-                                      ["string", ["one", "three"]],
-                                    ]),
-                   table1.join(table2, "key"))
+      expected = Arrow::Table.new([
+                                    ["key", [1, 3]],
+                                    ["number", [10, 30]],
+                                    ["key", [1, 3]],
+                                    ["string", ["one", "three"]]
+                                  ])
+      assert_equal(expected, table1.join(table2, "key"))
+      assert_equal(expected, table1.join(table2))
     end
 
     test("keys: Symbol") do
       table1 = Arrow::Table.new(key: [1, 2, 3],
                                 number: [10, 20, 30])
       table2 = Arrow::Table.new(key: [3, 1],
                                 string: ["three", "one"])
-      assert_equal(Arrow::Table.new([
-                                      ["key", [1, 3]],
-                                      ["number", [10, 30]],
-                                      ["key", [1, 3]],
-                                      ["string", ["one", "three"]],
-                                    ]),
-                   table1.join(table2, :key))
+      expected = Arrow::Table.new([
+                                    ["key", [1, 3]],
+                                    ["number", [10, 30]],
+                                    ["key", [1, 3]],
+                                    ["string", ["one", "three"]]
+                                  ])
+      assert_equal(expected, table1.join(table2, :key))
+      assert_equal(expected, table1.join(table2))
```

Comment:
This is redundant. Because it's the same assertion as the above newly added assertion.